ui/components/issuessection/issuessection.go

```diff
@@ -119,6 +119,7 @@ func (m Model) Update(msg tea.Msg) (section.Section, tea.Cmd) {
 					currIssue.Assignees.Nodes = removeAssignees(currIssue.Assignees.Nodes, msg.RemovedAssignees.Nodes)
 				}
 				m.Issues[i] = currIssue
+				m.Table.SetIsLoading(false)
```

Comment:
If `L122` and `L136` is not added, issues can not be shown because `Loading...` is showed always.